subprojects/docs/src/snippets/cpp/basic/kotlin/build.gradle.kts

```diff
@@ -56,15 +56,3 @@ application {
     targetMachines.set(listOf(machines.windows.x86, machines.windows.x86_64, machines.macOS.x86_64, machines.linux.x86_64))
 }
 // end::cpp-select-target-machines[]
```

Comment:
Ideally we would also turn the "application" part into a subproject and remove the root build script.

Thies does not need to be done as part of this PR though.

Are there many cases like this in the snippets? Where the root project is a "real" project and also has subprojects?